### PR TITLE
docs(intro): Document alternative way to import plugins

### PR DESCRIPTION
**Pull request by tmorehouse**
## Description of Pull Request:

Document alternative way to import individual Plugins

## PR checklist:

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Enhancement to an existing feature
- [ ] ARIA accessibility
- [x] Documentation update
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact:

**The PR fulfills these requirements:**

- [x] It's submitted to the `dev` branch, **not** the `master` branch
- [ ] When resolving a specific issue, it's referenced in the PR's title (i.e.
      `fixes #xxxx[,#xxxx]`, where "xxxx" is the issue number)
- [x] The PR should address only one issue or feature. If adding multiple features or fixing a bug
      and adding a new feature, break them into separate PRs if at all possible.
- [x] PR titles should following the
      [**Conventional Commits**](https://www.conventionalcommits.org/) naming convention (i.e.
      "fix(alert): not alerting during SSR render", "docs(badge): Updated pill examples, fix typos",
      "chore: fix typo in docs", etc). **This is very important, as the `CHANGELOG` is generated
      from these messages.**

**If new features/enhancement/fixes are added or changed:**

- [ ] Includes documentation updates (including updating the component's `package.json` for slot and
      event changes)
- [ ] New/updated tests are included and passing (if required)
- [ ] Existing test suites are passing
- [ ] The changes have not impacted the functionality of other components or directives
- [ ] ARIA Accessibility has been taken into consideration (does it affect screen reader users or
      keyboard only users? clickable items should be in the tab index, etc)

**If adding a new feature, or changing the functionality of an existing feature, the PR's
description above includes:**

- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a
      suggestion issue first and wait for approval before working on it)

### CODE REVIEW

File: docs/markdown/intro/README.md [hunk 1]
```diff
@@ -253,6 +255,24 @@ import { Scrollspy } from 'bootstrap-vue/es/directives'
 Vue.use(Scrollspy)
 ```
 
+Alternatively, you can import Plugins as follows:
+
+```js
+import BootstrapVue from 'bootstrap-vue'
+
+// This enables all the layout components such as <b-container>, <b-row>, <b-col>:
```
Comment (jackmu95) [inline docs/markdown/intro/README.md]: I think we can remove the trailing `:` in the comments. We don't use them at other places in comments.

File: docs/components/importdoc.vue [hunk 2]
```diff
@@ -128,9 +128,13 @@ export default {
...
     pluginImportCode() {
       const pluginLocation = this.isComponentRoute ? 'components' : 'directives'
+      const pluginDotted = this.isComponentRoute ? 'componentPlugins' : 'directivePlugins'
       return [
         `import { ${this.pluginName} } from 'bootstrap-vue/es/${pluginLocation}'`,
-        `Vue.use(${this.pluginName})`
+        `Vue.use(${this.pluginName})`,
+        '// or',
+        "import BoostrapVue from 'bootstrap-vue'",
+        `Vue.use(BootstrapVue.${pluginDotted}.${this.pluginName}`
```
[the remark this answers is not in the document]
Comment (tmorehouse) [reply docs/components/importdoc.vue]: Fixed

File: docs/markdown/intro/README.md [hunk 3]
```diff
@@ -236,23 +238,41 @@ You can also import component groups and directives as Vue plugins by importing
...
 import { Card } from 'bootstrap-vue/es/components'
 Vue.use(Card)
 
-// This imports directive v-b-scrollspy as a plugin:
+// This imports directive v-b-scrollspy as a plugin
 import { Scrollspy } from 'bootstrap-vue/es/directives'
 Vue.use(Scrollspy)
 ```
 
+Alternatively, you can import Plugins as follows:
```
Comment (pi0) [inline docs/markdown/intro/README.md]: There is no reason to introduce such alternative IMO. This breaks tree shake!

Comment (tmorehouse) [reply docs/markdown/intro/README.md]: Would destructuring BootstrapVue into componentPlugins and directivePlugins constants help?

Comment (tmorehouse) [reply docs/markdown/intro/README.md]: We could add named exports to main `index.js`, but that would probably break when using esm build (making people need to use `module.default`)?

Comment (pi0) [reply docs/markdown/intro/README.md]: @tmorehouse  Yes. destructuring + named exports should work but would be better to first test such usage in next RC before adding to the docs as an official way.

Comment (tmorehouse) [reply docs/markdown/intro/README.md]: OK... we can revert the changes for the doc additions.

Comment (tmorehouse) [reply docs/markdown/intro/README.md]: i.e. 
```js
import BootstrapVue from 'bootstrap-vue'
const { componentPlugins: { Layout, Modal } } = BoostrapVue

Vue.use(Layout)
Vue.use(Modal)
```

**Pull request merged**